packages/daimo-api/src/shovel/watcher.ts

```diff
@@ -107,13 +108,8 @@ export class Watcher {
 
   async getShovelLatest(): Promise<bigint> {
     const result = await this.pg.query(
-      `
-      select max(num) as num
-      from shovel.task_updates
-      where chain_id = $1
-      and backfill = false;
-    `,
-      [chainConfig.chainL2.id]
+      `select num from shovel.latest where src_name = $1 `,
```

Comment:
I think the `src_name` filter is not necessary since we've separated testnet/stage and mainnet/prod shovels?